src/lib/eventToApi.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import type { Event } from '../types';
+
+export type EventAPIType = {|
+  coreid: ?string,
+  data: ?Object,
+  published_at: Date,
+  ttl: number,
+  |};
```

Comment:
not sure if this is github formatting or not but it looks like you can remove a space on line 10